opwen_email_server/actions.py

```diff
@@ -238,6 +239,38 @@ def _new_email_id(cls, email: str) -> str:
         return sha256(email.encode('utf-8')).hexdigest()
 
 
+class ProcessServiceEmail(_Action):
```

Comment:
Given that this new "services via email" functionality is fairly separate from the core email service, I'd recommend to move all the code specific to it to a new module, like `opwen_email_server.mailers` or something like that. This will enable us to logically separate the functionality such that people who're interested in implementing a new mailer service only have to look into one module as opposed to the entire codebase.

If we introduce a new module, what do you think about structuring the mailers in a way similar to this to aid in code re-use and making mailer implementations look similar?

```
opwen_email_server
└── mailers
    ├── shared.py
    ├── echo
    │   ├── actions.py
    │   ├── integration.py
    │   └── format.py
    └── wikipedia
        ├── actions.py
        ├── integration.py
        └── format.py
```

Reply:
Good idea! This'll definitely be much easier for anyone looking to add a new service!